test/helpers/faker.js

```diff
@@ -1,12 +1,21 @@
 const faker = require('faker');
```

Comment:
Maybe we can use factory.build on test for every model, we can customize it passing by parameter the attributes we want to replace to the generated. Also when using build, we are avoiding create it on the db and we can verify the object with schemas and avoid a new dependency :smiley: